swift/ios-shoppe-demo/Resources/ShoppeItems.js

```diff
@@ -0,0 +1,114 @@
+[
```

Comment:
I've added an ID parameter to my mock data set: https://github.com/fullstorydev/react-shoppe-demo/blob/master/public/data/products.json

You can just replace this file with that. Also, this is a `.json` file, not a `.js` file.

Reply:
Perfect I'll knock that out!